draft-gpew-priv-ppm.md

```diff
@@ -1129,17 +1127,21 @@ duration. This could be made more clear.]
 
 Next, the leader extracts its own report shares and computes the initial
 preparation state for each as described in {{prep-start}}. It then removes from
-the report sequence any report for which the transition state is FAILED.
+the report sequence any report for which the transition state is FAILED. If the
+resulting sequence is empty, then the leader halts without issuing a request.
 
 Let `[aggregator]` denote the helper's API endpoint. The leader extracts the
 sequence of report shares for `[aggregator]` and sends sends a POST request to
 `[aggregator]/aggregate` with an Aggregate message of type `agg_init_req`
 and with AggregateInitReq as the payload.
 
 After verifying the authentication tag as described in
-{{aggregate-message-auth}}, the helper handles the AggregateInitReq by computing
-the initial preparation state for each ReportShare in `AggregateInitReq.seq` as
-described in {{prep-start}}. It then constructs an AggregateResp:
+{{aggregate-message-auth}}, the helper first checks that the nonces in
+`AggregateInitReq.seq` are all distinct. If two ReportShare messages have the
+same nonce, then the helper MUST abort with error "unrecognizedMessage". Next,
+the helper handles the AggregateInitReq by computing the initial preparation
```

Comment:
Yeah this is a very important point, but one we want to address throughout. Happy to address it here, if you like. In any case let's keep this in mind.

Reply:
I don't think it's worth making changes of that nature to the text in the interop branch because it'll make the eventual merge harder. Let's just keep this in mind.